frontend/src/utils/formatting/string.ts

```
@@ -0,0 +1,162 @@
+ *
+ * @example
+ * pluralize(1, 'apple'); // "1 apple"
+ * pluralize(3, 'apple'); // "3 apples"
+ * pluralize(2, 'child', 'children'); // "2 children"
+ *
+ * @since 1.0.0
+ * Generated by CodeRabbit
+ */
+export function pluralize(count: number, singular: string, plural?: string): string {
+  if (count === 1) return `${count} ${singular}`;
+  return `${count} ${plural || singular + 's'}`;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)